model/src/main/java/com/ge/predix/acs/rest/PolicyEvaluationRequestV1.java

```diff
@@ -43,6 +41,10 @@
 
     private LinkedHashSet<String> policySetsEvaluationOrder = EMPTY_POLICY_EVALUATION_ORDER;
 
+    public static LinkedHashSet<String> getEmptyPolicyEvaluationOrder() {
+        return EMPTY_POLICY_EVALUATION_ORDER;
```

Comment:
we are passing access to private variable through this function, do we really want to do this?